scripts/ci/stats/merged_prs.py

```diff
@@ -74,12 +74,19 @@ def process_pr(pr):
         prj['review_rule'] = "no"
 
 
+    created = pr.created_at
+    # if a PR was made ready for review from draft, calculate based on when it
+    # was moved out of draft.
```

Comment:
Comment should make it clear that it's "based on when it was *last* moved out of draft", since the code doesn't make it 100% clear this is what effectively happens if there were many ready/not-ready events.